### PR TITLE
Remove to update the same existing files from the repository by replacing it - with additional scripts 

### PR DESCRIPTION
**Pull request by djibal**
git commit -m "add some sub-module & subtree examples in and remove existing repo by replacing with this requested.git"

### CODE REVIEW

**Review by djibal — COMMENTED** [review]
Reviewed-all

**Review by djibal — COMMENTED** [review]
`Rev-viewed`: In "`Training-manuel`"

**Review by djibal — COMMENTED** [review]
Update

**Review by chrismisa — APPROVED** [review]

**Review by chrismisa — APPROVED** [review]

File: README [hunk 1]
```diff
@@ -0,0 +1 @@
+# Training-manual
```
Comment (chrismisa) [inline README:1]: OK

**Review by chrismisa — APPROVED** [review]

**Review by chrismisa — APPROVED** [review]

**Review by chrismisa — APPROVED** [review]

**Review by chrismisa — DISMISSED** [review]
OK

File: README (hunk 1 above)
Comment (melon-car) [reply README:1]: OK

**Pull request closed without merging**